src/components/EditorContainer.jsx

```diff
@@ -40,7 +40,6 @@ EditorContainer.propTypes = {
   source: PropTypes.string.isRequired,
   style: PropTypes.object.isRequired,
   onHide: PropTypes.func.isRequired,
-  onRef: PropTypes.func.isRequired,
 };
 
-export default EditorContainer;
+export default React.forwardRef(EditorContainer);
```

Comment:
This is a much nicer way to do what we need to do here, but it [doesn’t currently work with connected components](https://github.com/reduxjs/react-redux/issues/914), so I wasn’t able to do this refactor for `<Workspace>`